be/src/exec/cross_join_node.cpp

```diff
@@ -691,4 +700,15 @@ pipeline::OpFactories CrossJoinNode::decompose_to_pipeline(pipeline::PipelineBui
     return left_ops;
 }
 
+pipeline::OpFactories CrossJoinNode::decompose_to_pipeline(pipeline::PipelineBuilderContext* context) {
+    using namespace pipeline;
+
+    if (runtime_state()->enable_spill() && _join_op == TJoinOp::CROSS_JOIN) {
```

Comment:
should be supported in session var `spillable_operator_mask`, so that we can control this operator  easier